.github/workflows/pr.yaml

```diff
@@ -341,7 +341,7 @@ jobs:
       build_type: pull-request
       script: ci/cudf_pandas_scripts/run_tests.sh
   third-party-integration-tests-cudf-pandas:
-    needs: conda-python-build
+    needs: [conda-python-build, changed-files]
```

Comment:
This definitely was not silently failing for months 🤫

Reply:
:grimacing: